sampling_rate: 200

task_recorders:
  -
    class_name: JointStatesRecorder
    topic_name: /joint_states
splining_method: BSpline
